frontend/src/components/WofHeader.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <header class="header">
+    <div class="header__main-nav">
+      <img class="header__logo" src="../assets/logo.png" />
+      <router-link to="/" class="header__main-link">Wiki On Fire</router-link>
+    </div>
+    <div class="header__user-nav">
+      <router-link to="/login" class="header__nav-link" v-if="!user.length">
+        <span class="header__link-text">Sign In</span>
+        <wof-icon icon="wof-user" :size="2"></wof-icon>
+      </router-link>
+      <router-link to="/user/username" class="header__nav-link" v-if="user.length">
+        <span class="header__link-text">Username</span>
```

Comment:
Should it this be:
```suggestion
        <span class="header__link-text">{{ this.user }}</span>
```
?

Reply:
We will add this after user logging in